<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordChangeDialog</class>
 <widget class="QDialog" name="PasswordChangeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>密码修改</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>旧密码：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditOld"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>新密码：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditNew_3"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>确认密码：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditNewConfirm_3"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOK_5">
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>31</height>
        </size>
       </property>
       <property name="text">
        <string>确定(&amp;O)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOK_6">
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>31</height>
        </size>
       </property>
       <property name="text">
        <string>取消(&amp;O)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonOK_6</sender>
   <signal>clicked()</signal>
   <receiver>PasswordChangeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
